Sources/Aardvark/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyAccessedAPIType</key>
+			<string>NSPrivacyAccessedAPICategoryFileTimestamp</string>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>C617.1</string>
+			</array>
+		</dict>
+		<dict>
+			<key>NSPrivacyAccessedAPIType</key>
+			<string>NSPrivacyAccessedAPICategoryDiskSpace</string>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>7D9E.1</string>
+			</array>
+		</dict>
+		<dict>
+			<key>NSPrivacyAccessedAPIType</key>
+			<string>NSPrivacyAccessedAPICategoryUserDefaults</string>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>C56D.1</string>
+			</array>
+		</dict>
```

Comment:
I feel like we might be stretching the approved reason for `UserDefaults` here:
> Declare this reason if your third-party SDK is providing a wrapper function around user defaults API(s) for the app to use, and you only access the user defaults APIs when the app calls your wrapper function. This reason may only be declared by third-party SDKs. This reason may not be declared if your third-party SDK was created primarily to wrap required reason API(s).
Information accessed for this reason, or any derived information, may not be used for your third-party SDK’s own purposes or sent off-device by your third-party SDK.

The key/values are being sent off-device by email, but it's an explicit action by users as opposed to the SDK doing it behind the scenes. I noticed that the user defaults plist also contains keys/values set by the system in addition to app-specific defaults. Here's an example where only the bottom 3 are saved by the app:

![image](https://github.com/square/Aardvark/assets/2532658/c1ff6b85-7690-4320-922a-7d16f23d8817)

Would love to get people's thoughts on this

Reply:
yeah i'm not entirely sure about this use case. a conservative reading of the guidelines makes it seem like this might be a violation, but also there is explicit affirmation to submit the data and maybe technically the SDK isn't sending the data off-device (the user is by pressing send on the email? unless i'm misunderstanding the UX here...). i feel like you could argue either way for this...